Add node:logs command - closes #692

ManuGowda opened the pull request:
### What was the problem?
No implementation exists
### How did I fix it?
Implement the node logs feature
### How to test it?
`./bin/run node:logs --name=test-1.5`
### Review checklist

* The PR resolves #692 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

Commit:
:seedling: Implement node logs

shuse2 commented on src/commands/node/logs.ts:
```diff
@@ -0,0 +1,60 @@
+
+    const { pm2_env } = await describeApplication(name);
+    const { pm_cwd: installDir, LISK_NETWORK: network } = pm2_env as Pm2Env;
+    const { logFileName }: NodeConfig = getNetworkConfig(installDir, network);
```
No need to parse the type here

ManuGowda replied:
why not use the type? any specific reason?

shuse2 replied:
because it's returning the type `NodeConfig`, it's redundant

shuse2 commented on src/commands/node/logs.ts:
```diff
@@ -0,0 +1,60 @@
+
+    const { stderr, stdout } = childProcess.spawn('tail', ['-f', fName]);
+
+    stdout.on('data', (data) => {
```
nice solution.
shouldn't we still listen to close event and clean up the stream and child process?

ManuGowda replied:
hmmm I will investigate this point 👍

Commit:
:recycle: Tidy up after review comments

Handle close, error and disconnected events

:fire: Remove unwanted code

Revert ":fire: Remove unwanted code"

This reverts commit 73ee4976344f75dbdd57721d46dec387cb4e2651.

Revert ":fire: Remove unwanted code"

This reverts commit 73ee4976344f75dbdd57721d46dec387cb4e2651.

Commit:
:fire: Remove unwanted code

shuse2 commented on src/commands/node/logs.ts:
```diff
@@ -0,0 +1,70 @@
+		const { pm2_env } = await describeApplication(name);
+		const { pm_cwd: installDir, LISK_NETWORK: network } = pm2_env as Pm2Env;
+		const { logFileName } = getNetworkConfig(installDir, network);
+		const fName = `${installDir}/${logFileName}`;
```
```suggestion
		const fileName = `${installDir}/${logFileName}`;
```

Commit:
:recycle: Update variable name
